packages/ember-metal/lib/index.js

```diff
@@ -306,9 +306,19 @@ Ember.run = run;
 @for Ember
 @private
 */
-Ember.Backburner = Backburner;
-// this is the new go forward, once Ember Data updates to using `_Backburner` we
-// can remove the non-underscored version.
+Ember.Backburner = function() {
+  deprecate(
+    'Usage of Ember.Backburner is deprecated.',
+    false,
+    {
+      id: 'ember-metal.ember-backburner',
+      until: '2.8.0',
+      url: 'http://emberjs.com/deprecations/v2.x/#toc_ember-backburner'
+    }
+  );
+
+  return new Backburner(...arguments);
```

Comment:
Ya, when you are calling a method on an object and using `...arguments` it binds the `this` to that object:

```
// before
foo.bar(...arguments);

// after
foo.bar.apply(foo, arguments);
```

But apparently, when there is no obvious binding it uses `Function.prototype.bind`...